draft-trammell-taps-interface.md

```diff
@@ -494,7 +505,9 @@ requiring a checksum when receiving.
 
 ### Interface Type {#prop-intf-type}
 
-This enumerated property specifies which kind of access network interface,
+Type: Enumeration 
```

Comment:
An Enumeration is a set of (Type, Preference) tuples, correct? We might want to say that somewhere.

Reply:
Oh - that is not what I wanted to state here – my idea was just to use this as "prefer interface type", e.g., Wifi.

However, I like your Idea of making that a tuple much better.